specs/daito/Sushi_Fileshare_Design.md

```diff
@@ -0,0 +1,95 @@
+# Sushi Fileshare Design
+
+**Author(s)**: [jackhaibo](https://github.com/jackhaibo)
+
+## Summary
+
+This is the design spec for the CSI fileshare feature.
+
+## Motivation
+
+Currently kubernetes supports volume features, and fileshare features can also be added. 
+
+### Goals
+
+* Fileshare plugin is independent of volume plugin.
+* NFS driver and oceanstor 18000 v5 driver fileshare can be created, deleted, attached, detached in kubernetes cluster.
+ 
+### Non-Goals
+
+Fileshare replication can be implemented in kubernetes cluster in the next phrase. 
+ 
+## Design Details
+
+* CreateFileShare(req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error)
```

Comment:
Those are not CSI interfaces.  What's the relationship between CSI RPCs and these file share functions.  Please draw a diagram to illustrate the relationship between CSI RPCs and these file share interfaces and also how these functions interface with Hotpot file share REST APIs.